Task 1/Tower-of-Hanoi/tower_of_hanoi.cpp

```diff
@@ -0,0 +1,30 @@
+

```

Comment:
Tower of Hanoi exists already under the Task 3 folder, can we update this PR to include some other Algorithm/any other problem+solution pair under Task 1?